helpers/object.ts

```diff
@@ -0,0 +1,9 @@
+export const getPropertyRecursively = (property, data) => {
```

Comment:
`getPropertyRecursively` has to explicitly take in a type that and return that same type,

`export function getProp<T = any>(prop: string, data = {}): T[keyof T]`

If this isn't possible, then _all variables that use this function must explicitly say what their type is_